cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -4369,6 +4381,159 @@ msgstr "Chwiliwch am gyfryngwr teulu yn eich ardal chi i ganfod a ydych yn gymwy
 msgid "Find your local mediator"
 msgstr "Dewch o hyd i’ch cyfryngwr lleol"
 
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Accessibility statement"
+msgstr "Datganiad hygyrchedd"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Using this service"
+msgstr "Defnyddio’r gwasanaeth hwn"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The Legal Aid Agency (<abbr>LAA</abbr>) is responsible for the Civil Legal Advice (<abbr>CLA</abbr>) service."
+msgstr "Yr Asiantaeth Cymorth Cyfreithiol (<abbr>LAA</abbr>) sy’n gyfrifol am y gwasanaeth Cyngor Cyfreithiol Sifil (<abbr>CLA</abbr>)."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "We want as many people as possible to be able to use this service. For example, that means you should be able to:"
+msgstr "Rydyn ni am i gynifer o bobl â phosibl allu defnyddio’r gwasanaeth hwn. Er enghraifft, mae hynny’n golygu y dylech allu:"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "change colours, contrast levels and fonts"
+msgstr "newid lliwiau, lefelau cyferbyniad a ffontiau"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "zoom in up to 300&#37; without the text spilling off the screen"
+msgstr "chwyddo mewn hyd at 300&#37; heb i’r testun ollwng oddi ar y sgrin"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "navigate the service using just a keyboard"
+msgstr "symud o gwmpas gan ddefnyddio bysellfwrdd yn unig"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "AbilityNet has advice on making your device easier to use if you have a disability."
+msgstr "Mae gan AbilityNet gyngor ar wneud eich dyfais yn haws ei defnyddio os oes gennych chi anabledd."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "What to do if you can’t access part of this service"
```

Comment:
This was changed to `Feedback and contact information` on line 34 of the accessibility statement

Reply:
That is part of the new layout, things have been moved around a bit.

Reply:
That means there's no welsh for `Feedback and contact information` then. Coming later?

Reply:
Coming later....